vip_hci/metrics/roc.py

```diff
@@ -194,11 +194,14 @@ def compute_tpr_fps(self, npix=1, min_distance=1):
             x, y = self.list_xy[i]
 
             for m in self.methods:
-                res = compute_binary_map(m.probmaps[i], m.thresholds, x, y,
-                                         npix=npix, min_distance=min_distance)
-                m.detections.append(res[0])
-                m.fps.append(res[1])
-                m.bmaps.append(res[2])
+                dets, fps, bmaps = compute_binary_map(
+                    m.probmaps[i], m.thresholds, fwhm=self.dataset.fwhm,
+                    injections=(y, x), **kwargs
```

Comment:
All the functions in VIP that are exposed to the user, use the convention (X,Y). I've made this modification to compute_binary_map in a previous PR.

Reply:
👍